lib/tests/behat/behat_general.php

```diff
@@ -634,6 +634,57 @@ function($context, $args) {
 
     }
 
+    /**
+     * Checks, that page contains specified text or another. It also checks if the text is visible when running Javascript tests.
+     *
+     * @Then /^I should see "(?P<text_string>(?:[^"]|\\")*)" or "(?P<text_string2>(?:[^"]|\\")*)"$/
+     * @throws ExpectationException
+     * @param string $text
+     * @param string $secondtext
+     */
+    public function i_should_see_or($text, $secondtext) {
```

Comment:
@marcghaly I'm not sure about touching this ... as `lib/tests/behat/behat_general.php ` core behat library. Adding code here seems for me not right.

Reply:
please revert any changes here, this is core behat library

Reply:
moved to `mod/quiz/tests/behat/behat_mod_quiz.php`